lib/jobs/emc-redfish-catalog.js

```diff
@@ -56,27 +58,155 @@ function EmcRedfishCatalogJobFactory(
      */
     EmcRedfishCatalogJob.prototype._run = function() {
         var self = this;
-        return self.catalogElements()
+        return Promise.resolve(self.chassis)
+        .each(self.catalogElements.bind(self))
```

Comment:
@benbp Probably not too much difference as-is, typically the chassis list size is 1 but I could see a benefit as the chassis/catalog grows.